packages/api-explorer/src/lib/Oas.js

```diff
@@ -5,8 +5,7 @@ class Operation {
   constructor(oas, path, method, operation) {
     Object.assign(this, operation);
     this.oas = oas;
-    // strips trailing slash
-    this.path = path.endsWith('/') ? path.slice(0, -1) : path;
```

Comment:
Surprised there was no test for this in Oas.test.js?

Reply:
There was! I removed it here: https://github.com/readmeio/api-explorer/pull/214/files#diff-118b2290dc906ad945edd96c4401c538L16